fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -1 +1,12 @@
-The reasonable Tox client for Android.
+<p><b>aTox, a beautiful and modern Tox client.</b></p>
+
+<p>aTox is a free open-source messaging app for the Tox protocol.</p>
+
+<p><b>Why aTox?</b></p>
+<ul>
+<li>Open-source: free to share, inspect, and change as you like.</li>
+<li>Reasonable: sane defaults, no guesses about what you care about, doesn't ignore what <b>you</b> want.</li>
+<li>End-to-end encrypted: the only people who can see your conversations are the people you're talking to.</li>
+<li>Distributed: no central servers that can be taken down or forced to turn over data.</li>
```

Comment:
s/forced/compelled

Reply:
Fixed. Also added links both to the aTox GitHub and https://tox.chat for more info about the app and the protocol like you suggested on IRC.